README.md

```diff
@@ -1,2 +1,28 @@
-# autopsy-packager
+# autopsy-packager ![Github Workflows](https://github.com/labcif/autopsy-packager/actions/workflows/packaging.yml/badge.svg?branch=main)
```

Comment:
I'd prefer if it were before or after the title.

```md
# autopsy-packager
![Github Workflows](https://github.com/labcif/autopsy-packager/actions/workflows/packaging.yml/badge.svg?branch=main)
```

Reply:
Sure, done. I added it right after the first heading.